src/dc_mimeparser.rs

```diff
@@ -1422,6 +1422,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
```

Comment:
why, this should be a very fast test?

Reply:
if not, we should figure out very fast why not

Reply:
didn't want to think: i get a warning that this test is running more than 60secs. feel free to close. am currently to sort through rustifying the incoming code path, especially decyrption. (@adb had a strange unwrap() issue there, besides it being super hard to understand)

Reply:
uff, this runs more than 60 seconds Oo that sounds like sth is very wrong

Reply:
I figured it out, the test that is slow is not this one, it is `dc_mimeparser::tests::test_dc_mailmime_parse_crash_fuzzy`, and it is slow because it runs throuhg a lot of variations in the proptest